.github/workflows/linux_build.yml

```diff
@@ -11,6 +11,7 @@ on:
     branches:
       - master
       - papyrus
+      - papyrus_product
```

Comment:
This is a temporary development branch, right? So, it shouldn't be in this list.